### PR TITLE
Init new website

### PR DESCRIPTION
**Pull request by CAnorbo**

### CODE REVIEW

**Review by luzius1089 — COMMENTED** [review]
L very GTM! 

- we do not have docs.gingerwallet.io
- add link to GitHub at the bottom of the website
- API is at https://api.gingerwallet.io/swagger

File: wwwroot/404.html [hunk 1]
```diff
@@ -4,55 +4,75 @@
...
-    <meta name="title" content="Wasabi Wallet: 404 Page not found">
-    <meta name="description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for desktop, that implements trustless coinjoin over the Tor anonymity network.">
-    <meta name="keywords" content="wasabi wallet,wasabiwallet,wasabi bitcoin wallet,wassabi wallet,bitcoin wasabi wallet,wasabiwallet,download wasabi wallet,wasabi btc,wasabi wallet,zksnacks,wallet wasabi,wasabi wallet fees,what is wasabi wallet,coinjoin, coin join">
-    <meta http-equiv="onion-location" content="http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion" />
+    <meta name="title" content="Ginger Wallet: 404 Page not found">
+    <meta name="description" content="Ginger is an open-source, non-custodial, privacy-focused Bitcoin wallet for desktop, that implements trustless coinjoin over the Tor anonymity network.">
+    <meta name="keywords" content="Ginger wallet,Gingerwallet,Ginger bitcoin wallet,wassabi wallet,bitcoin Ginger wallet,Gingerwallet,download Ginger wallet,Ginger btc,Ginger wallet,wallet Ginger,Ginger wallet fees,what is Ginger wallet,coinjoin, coin join">
```
[the remark this answers is not in the document]
Comment (luzius1089) [reply wwwroot/404.html:9]: or secure coinjoin

File: wwwroot/index.html [hunk 2]
```diff
@@ -1,162 +1,404 @@
...
+                                    After years of privacy research and development, we’ve realized that having good tools is not enough—they must also be easy to use. Otherwise, they will only be utilized by a minority, making those users stand out. To encourage the adoption of these tools, Ginger is designed to be user-friendly and to provide privacy automatically by default.
                                 </p>
                             </div>
+                            <ul class="timeline">
+                                <li>
+                                    <div class="timeline-title">2024 - <img src="img/ww_logo.svg" class="timeline-small-img" alt="Ginger Wallet"> Wasabi Wallet without coinjoin</div>
+                                    <p>The default coinjoin coordinator of Wasabi Wallet stopped operating.</p>
+                                </li>
+                                <li>
+                                    <div class="timeline-title">2024 - <img src="img/ginger_logo_small.svg" class="timeline-small-img" alt="Ginger Wallet"> Ginger WALLET</div>
```
Comment (soosr) [inline wwwroot/index.html:101]:
```suggestion
                                    <div class="timeline-title">2024 - <img src="img/ginger_logo_small.svg" class="timeline-small-img" alt="Ginger Wallet"> Ginger Wallet</div>
```

**Review by soosr — COMMENTED** [review]
**Asked ChatGPT for bg colors, here are the answers:**

Dark Slate Grey (#2F4F4F): This color provides a strong contrast with white text and pairs well with your accent color.

Charcoal (#36454F): A slightly lighter and warmer tone compared to your current background, which maintains readability and offers a modern look.

Deep Navy (#1B263B): This dark blue provides a nice contrast without being too harsh, and it can give a sophisticated touch to your webpage.

Ebony (#0C0B1A): A near-black color with a hint of blue, offering a subtle variation from your current background while keeping the overall dark theme.

Gunmetal (#2C3539): A dark, muted grey that provides excellent contrast for white text and a complementary tone for your accent color.

**Non blueish:**

Dark Olive (#3B3B2E): This color has an earthy feel and pairs nicely with your accent color A68D67.

Charcoal (#36454F): A dark, neutral grey that maintains readability and gives a modern look without any blueish tint.

Dark Brown (#2E1B0B): This rich, dark brown offers a warm and inviting contrast to white text and works well with your accent color.

Smoky Black (#1A1A1A): A deep black with a slightly warmer undertone compared to pure black, ensuring good contrast and a cohesive look with your accent color.

Raisin Black (#242124): A very dark grey with subtle hints of brown, providing a sophisticated and elegant backdrop.

##

**Are any of these works better? I couldn't test.**

**Review by soosr — APPROVED** [review]

**Pull request merged**